### PR TITLE
Master

### PR DESCRIPTION
**Pull request by nkozyra**

### CODE REVIEW

File: src/api_v1.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+func API_Page() {
+
+}
+
+func API_Topic() {
+
+}
+
+func API_Forum() {
```
Comment (seangibat) [inline src/api_v1.go:17]: love it.

File: templates/register.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <div class="mdl-textfield mdl-js-textfield">
+        <input class="mdl-textfield__input" type="email" pattern="*{3,}" id="user_email" name="user_email">
+        <label class="mdl-textfield__label" for="user_email">Email</label>
+        <span class="mdl-textfield__error">Must be valid email</span>
+      </div>
+      <div class="mdl-textfield mdl-js-textfield">
+        <input class="mdl-textfield__input" type="password" pattern="*{6,12}" id="user_pass" name="user_pass">
+        <label class="mdl-textfield__label" for="user_pass">Password</label>
+        <span class="mdl-textfield__error">Must be 6-12 characters</span>
```
Comment (seangibat) [inline templates/register.html:25]: Wait, it can't be longer than 12? Why?

File: src/api_v1.go (hunk 1 above)
Comment (aazlant) [inline src/api_v1.go:17]: Fuck. Yes.

File: templates/register.html (hunk 2 above)
Comment (nkozyra) [inline templates/register.html:25]: it could, but now the template won't validate it to be 6-12 chars

**Pull request merged**